lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -729,6 +729,16 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
     if (!DC)
       return TheCU;
 
+    auto createContext = [&](NominalTypeDecl &NTD) {
+      GenericContextScope scope(
```

Comment:
This is unused?

Reply:
That's an RAII object.

Reply:
It temporarily modifies IGM's TypeConverter.